docs/en/latest/deployments/kind.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Install Ingress APISIX on Kind
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+This document explains how to install Ingress APISIX on [kind](https://kind.sigs.k8s.io/).
+
+## Prerequisites
+
+* Install [kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation).
+* Install [Helm](https://helm.sh/).
+
+## Create Cluster
+
+Create a kind cluster with `extraPortMappings` and `node-labels`. You can click this [link](https://kind.sigs.k8s.io/docs/user/ingress/#create-cluster) for more information.
+
+Command as follows for reference:
+
+```shell
+cat <<EOF | kind create cluster --config=-
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  kubeadmConfigPatches:
+  - |
+    kind: InitConfiguration
+    nodeRegistration:
+      kubeletExtraArgs:
+        node-labels: "ingress-ready=true"
```

Comment:
IIRC, we don't need these configurations.

Reply:
Yeah, we just use `kind create cluster`.
And then run as follows:
```shell
helm repo add apisix https://charts.apiseven.com
helm repo add bitnami https://charts.bitnami.com/bitnami
helm repo update
kubectl create ns ingress-apisix
helm install apisix apisix/apisix \
  --set gateway.type=NodePort \
  --set ingress-controller.enabled=true \
  --namespace ingress-apisix \
  --set ingress-controller.config.apisix.serviceNamespace=ingress-apisix
kubectl get service --namespace ingress-apisix
```

Everything are done.

I will delete these configurations.